CMakeLists.txt

```diff
@@ -1199,6 +1201,21 @@ if(NOT SKIP_INSTALL_HEADERS AND NOT SKIP_INSTALL_ALL)
 endif()
 if(NOT SKIP_INSTALL_FILES AND NOT SKIP_INSTALL_ALL)
     install(FILES ${ZLIB_PC} DESTINATION "${PKGCONFIG_INSTALL_DIR}")
+    # INFO: Mimics official zlib CMake target
+    # Generates ZLIB.cmake in case ZLIB_COMPAT=ON and always exports the CMake target ZLIB::ZLIB
+    # In case ZLIB_COMPAT=OFF, the CMake target and file follows zlib-ng naming convention
+    if (ZLIB_COMPAT)
+        if (TARGET zlib)
+            set_target_properties(zlib PROPERTIES EXPORT_NAME ZLIB)
+        else()
+            set_target_properties(zlibstatic PROPERTIES EXPORT_NAME ZLIB)
+        endif()
+    endif()
+    install(TARGETS ${ZLIB_INSTALL_LIBRARIES}
```

Comment:
This doesn't work with some cmake versions, missing `DESTINATION` parameter.

Reply:
@mtl1979 good catch, indeed I only tested with newer version of cmake.

I tested now with cmake 3.6.3 and the PR was broken due the DESTINATION missing in the install(). I added it now and is working. Thank you!

Reply:
I mostly use cmake 3.10.2 on Ubuntu 18.04 for testing incompatibility issues... It's pretty much the most recent cmake version that still doesn't support all the basic features we use in testing zlib-ng. I used to have Ubuntu 16.04 with even older cmake version but I couldn't install any updates or new packages as the repositories were taken offline.

Reply:
@mtl1979 I run a docker image with Ubuntu 16.04 + CMake 3.6.3 (installed via pip).

Reply:
I used to use Docker until it became obvious it's just waste of disk space... Now I just launch Ubuntu straight from disk images as my CPU can do that without additional software.